clients/python/setup.py

```diff
@@ -5,12 +5,14 @@
 
 setuptools.setup(name='myanmartools',
```

Comment:
Add some [classifiers](https://packaging.python.org/tutorials/packaging-projects/#classifiers)?

Reply:
I think the main use of classifiers is for browsing projects on PyPI. [Here](https://pypi.org/classifiers/) is the list of classifiers, but they are not defined so they create a bit of ambiguity e.g. [when should "Natural Language" classifiers be used](https://stackoverflow.com/questions/33184174/when-should-the-natural-language-pypi-classifier-be-used) (I was hoping to find Myanmar/Burmese but did not). Also, in "Operating Systems", "Linux" is listed under "POSIX", and "UNIX" is a separate one.

If you would like to add classifiers, you can add
```python
classifiers=[
    'Development Status :: 4 - Beta',
    'Intended Audience :: Developers',
    'Intended Audience :: Science/Research',
    'License :: OSI Approved :: Apache Software License',
    # numpy claims to support Windows, POSIX, Unix and MacOS
    # but might be problematic with Android and iOS
    'Operating System :: OS Independent',
    'Programming Language :: Python :: 3.7',
    'Programming Language :: Python :: 3.8',
    'Topic :: Text Processing'
]
```

Reply:
Great; can you add them here to this PR?

Seems like the ones about "Programming Language" are misleading, since we should remain compatible with future versions of Python.  Seems like saying "Python 3.7 or 3.8" would get stale.

Reply:
Yes, now Python 3.9 is still in alpha, but when the stable release comes out, it needs to be updated.